### ISSUE TITLE
testcafe reporter cannot be published on npm

### ISSUE THREAD

**Issue by hdorgeval** [DevExpress/testcafe#2382]
### Are you requesting a feature or reporting a bug?
I am currently developing a testcafe reporter [testcafe-reporter-cucumber-json](https://github.com/hdorgeval/testcafe-reporter-cucumber-json).

### What is the current behavior?
The publish-please returns vulnerable dependencies:
```sh
ERRORS
  * (+) 1 vulnerabilities found
 Name     Installed   Patched    Path      More Info
 lodash   4.16.4      >=4.17.5   testcafe-reporter-cucumber-json@0.3.1 > testcafe@0.19.2 > testcafe-hammerhead@13.3.1 > lodash@4.16.4   https://nodesecurity.io/advisories/577
```

The only way to publish to npm is to deactivate vulnerability checking in publish-please configuration file.

These errors are also reported by nsp when doing an npm install just after cloning the testcafe repo:
```text
nsp check --reporter checkstyle
(+) 1 vulnerability found
┌────────────┬────────────────────────────────────────────────────────────────────┐
│            │ Prototype Pollution                                                │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Name       │ lodash                                                             │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ CVSS       │ 2 (Low)                                                            │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Installed  │ 4.16.4                                                             │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Vulnerable │ <4.17.5                                                            │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Patched    │ >=4.17.5                                                           │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Path       │ testcafe@0.20.0-alpha.3 > testcafe-hammerhead@13.4.3 >             │
│            │ lodash@4.16.4                                                      │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ More Info  │ https://nodesecurity.io/advisories/577                             │
└────────────┴────────────────────────────────────────────────────────────────────┘
```
### What is the expected behavior?
testcafe and testcafe-hammerhed should not reference any dependency with known vulnerabilities.

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:

Test code

```js

```

### Specify your

* operating system: Mac OS X
* testcafe version: 0.19.2
* node.js version: v9.2.0 and npm v5.5.1

**Comment by kirovboris** [comment]
Hi @hdorgeval, 

Thank you for reporting it. We'll fix it ASAP, this fix will be included in `testcafe@0.20.0`. If it's necessary we can provide the dev version as well.

**Comment by miherlosev** [comment]
Hi @hdorgeval 

The fix was published in `testcafe@0.20.0-alpha.4`
